blockchain/v0/reactor_test.go

```diff
@@ -70,9 +70,10 @@ func newBlockchainReactor(
 
 	blockDB := dbm.NewMemDB()
 	stateDB := dbm.NewMemDB()
+	sstore := sm.NewStateStore(stateDB)
```

Comment:
I think we should write this out as `stateStore` everywhere instead of `sstore`, similar to what we do for `blockStore` - it's easier to read, and we only save a measly 4 characters anyway.